packages/next/src/migrations/update-17-2-7/remove-eslint-rules-patch.ts

```diff
@@ -0,0 +1,18 @@
+import { Tree, formatFiles, getProjects } from '@nx/devkit';
+import { updateOverrideInLintConfig } from '@nx/eslint/src/generators/utils/eslint-file';
+
+export default async function update(tree: Tree) {
+  const projects = getProjects(tree);
+  projects.forEach((project) => {
+    updateOverrideInLintConfig(
+      tree,
+      project.root,
+      (o) =>
+        o.rules?.['@next/next/no-html-link-for-pages'] &&
+        o.files.includes('**/*.*'),
```

Comment:
`o.files?.`

Reply:
Would be a broken config on the user's side but may as well 🤷